addon/styles/addon.scss

```diff
@@ -74,7 +74,7 @@ $frost-modal-overflow-shadow-color: rgba(0, 0, 0, .13);
       margin-left: 10px;
     }
 
-    &-text {
```

Comment:
I'm 👍 for a minor here - I think leaving it as text is confusing and I don't believe anyone has targeted it yet (you would be the only one @sandersky)